index.bs

```diff
@@ -2277,6 +2302,193 @@ create them does not matter.
  1. Return « |branch1|, |branch2| ».
 </div>
 
+<div algorithm>
+ <dfn abstract-op lt="ReadableByteStreamTee"
+ id="readable-byte-stream-tee">ReadableByteStreamTee(|stream|)</dfn>
```

Comment:
The fact that cloneForBranch2 is ignored (and treated as always-true) for byte streams is interesting and worth noting carefully, probably in the web-developer facing domintro for tee().

Or, would it be worthwhile and possible to respect it? Probably not given all the transfers...

Reply:
Correct, both `ReadableByteStreamController.enqueue()` and `ReadableStreamBYOBRequest.respond()` transfer the buffer. So it's impossible to pass the same chunk to both branches, we *must* copy it first.

I'll try to write something for the domintro. 🙂

Reply:
Added a note, have a look. 😉